server/test/controllers/auth.test.ts

```diff
@@ -60,7 +60,7 @@ describe('Test POST /v1/auth/signup - create a new user', () => {
         name: 'aalto',
         password: 'grades',
         studentID: '123456',
-        role: 'SYSADMIN'
+        role: 'ADMIN'
```

Comment:
Use the `enum` instead? Also applies to the other `'ADMIN'` change in this file.